libraries/lib-dynamic-range-processor/DynamicRangeProcessorUtils.h

```diff
@@ -0,0 +1,71 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  DynamicRangeProcessorUtils.h
+
+  Matthieu Hodgkinson
+
+**********************************************************************/
+#pragma once
+
+#include "DynamicRangeProcessorTypes.h"
+#include "Internat.h"
+#include "TranslatableString.h"
+
+namespace DynamicRangeProcessorUtils
+{
+template <typename Struct> struct Preset
+{
+   const TranslatableString name; // Not sure this needs to be translat
```

Comment:
typo: "translated" ?

Reply:
I wanted to get rid of that comment: now I know it should be translatable.